middlewares/accesslog/logger.go

```diff
@@ -36,6 +36,7 @@ type LogHandler struct {
 	file     *os.File
 	filePath string
 	mu       sync.Mutex
+	redact   map[string]int
```

Comment:
If this is just supposed to represent a set, then I'd use `bool` as value type. I _think_ this is also what the stdlib does in similar cases.